backend/src/transaction/dto/create-transaction-request.dto.ts

```diff
@@ -44,9 +47,9 @@ export class CreateTransactionRequestDto {
   @ApiProperty({
     description: 'A list of public keys to be used for signing the transaction',
     example: [
-      '75ec8c1997089874ce881690e95900f821a7f69152814728be971e67e4bc2224',
-      '4617e0079f0e943fc407e77ca9fc366f47ccdb4cbec6d5d51eeb996e781c052d',
-      'a0d7a883021253dc9f260ca7934b352f2d75e96d23ebdd1b3851ec0f0f0729d1',
+      '0x75ec8c1997089874ce881690e95900f821a7f69152814728be971e67e4bc2224',
```

Comment:
Why do we include "0x"? in hedera (sdk for example) they never use 0x. I like it more with 0x because its more explicit saying "this string is representing bytes in hexadecimal" but its not a good example if the client (stablecoin) is using the sdk and its going to have the public keys in hex without the "0x" or the DER keys 🤔